### DISCUSSION TITLE
Is there a site where I can preview the yaml file?

### DISCUSSION THREAD

**Discussion by sunyeongan** [euphony-io/euphony#144] [Q&A] [answered]
https://github.com/euphony-io/euphony/issues/137
I modified feature_request.md 
But I have a question. Is there a site where I can preview the yaml file like a markdown preview?

I found was a site that solved syntax errors in the yaml file. (https://jsonformatter.org/yaml-viewer)
but I want a site where I can see the preview like the following site. (https://dillinger.io/)
Below is the yaml file that I wrote!...

---
 name: Feature request
 about: Suggest an idea for this project
 title: ''
 labels: ''
 assignees: ''

---
 body:
  - type: markdown
    attributes:
      value: Is your feature request related to a problem? Please describe.
  - type: textarea
    attributes:
       label: Describe the feature you'd like to request
       description : A clear and concise description of what the problem is. Ex.  I'm always frustrated when [...]
    validations:
      required: true
  - type: textarea
    attributes:
       label: Describe the solution you'd like
       description: A clear and concise description of what you want to happen.
     validations:
       required: true
  - type: textarea
    attributes:
       label: Describe alternatives you've considered
       description: A clear and concise description of any alternative solutions  or features you've considered.
    validations:
      required: true
  - type: textarea
    attributes:
       label: Additional context
       description: Add2 any other context or screenshots about the feature       request here.
    validations:
      required: true

**Comment by yeonns** [comment, marked as answer]
@sunyeongan Oh, I also want to know yaml preview for issue template! 😮 
The easiest way is like below...

1. Fork ```euphony-io/euphony``` -> In my case, forked repository will be ```yeonns/euphony```.
2. Activate ```issue``` in your forked repository settings. (Settings - General - Features - Issues)

![image](https://user-images.githubusercontent.com/10149398/178738979-8363ee33-b61b-4071-a971-24544784356a.png)

3. Make issue template file (ex. ```.github/ISSUE_TEMPLATE/sample-app-proposal.yaml```) and commit to your forked repository
4. You can check if the file is applied at ```Issues - New issue```
(I made ```test.yaml``` in my repository, and it is like below. you can check my [repository](https://github.com/yeonns/euphony))

![image](https://user-images.githubusercontent.com/10149398/178739795-21c8e709-565a-4c6d-81f8-dd9106ff6553.png)

5. Or, after commit file, you can check it with accessing file. Please check my [link](https://github.com/yeonns/euphony/blob/master/.github/ISSUE_TEMPLATE/test.yaml)

![image](https://user-images.githubusercontent.com/10149398/178740546-de2a2cf5-d6d3-4c70-825d-d2597e417867.png)

**Reply by designe** [reply]
Super detailed with example! 👍

**Reply by sunyeongan** [reply]
Thank you for your kind explanation 
I understood. I'll try it soon.
